website/templates/public/pages/faq.mako

```diff
@@ -117,8 +117,8 @@
 <p>If you’ve already claimed the account (and now have two OSF accounts!), send us an <a href="mailto:support@osf.io">email</a>. We can merge the accounts for you so that you’ll only have one - all of your projects will be under one account, your name will only show in search once, and you will have one login email and password.</p> 
 <p>If the project is private, claim the account, and then follow the steps above.</p>
 
-<h3>How do I make folders to organize the files in my project?</h3>
-<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
+<h3>What is the difference between a component and a folder?</h3>
+<p>A folder can be used to organize files within a project or component - just like a folder on your own computer groups files together. A component also groups files together, but has added functionality. Components have privacy and sharing settings independent of the overarching project's settings, their own unique, persistent identifiers for citation, and their own wiki and add-ons.</p>
```

Comment:
I'd be okay with that change.

On Mon, May 11, 2015 at 11:56 AM, Steven Loria <notifications@github.com>
wrote:

> In website/templates/public/pages/faq.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/2662#discussion_r30052120>
> :
>
> > @@ -117,8 +117,8 @@
> >  <p>If you’ve already claimed the account (and now have two OSF accounts!), send us an <a href="mailto:support@osf.io">email</a>. We can merge the accounts for you so that you’ll only have one - all of your projects will be under one account, your name will only show in search once, and you will have one login email and password.</p>
> >  <p>If the project is private, claim the account, and then follow the steps above.</p>
> >
> > -<h3>How do I make folders to organize the files in my project?</h3>
> > -<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
> > +<h3>What is the difference between a component and a folder?</h3>
> > +<p>A folder can be used to organize files within a project or component - just like a folder on your own computer groups files together. A component also groups files together, but has added functionality. Components have privacy and sharing settings independent of the overarching project's settings, their own unique, persistent identifiers for citation, and their own wiki and add-ons.</p>
>
>  A component also groups files together...
>
> I'm not sure that we want to encourage users to think of components as
> "file folders + added functionality". Components seem to be conceptually
> distinct from file folders. As in #2689
> <https://github.com/CenterForOpenScience/osf.io/pull/2689>, I would
> suggest describing components "like subprojects".
>
> open for feedback @JeffSpies <https://github.com/JeffSpies> @bnosek
> <https://github.com/bnosek>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/2662/files#r30052120>
> .
>